frontend/pages/[gameId]/postgameplayer.tsx

```diff
@@ -0,0 +1,109 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { PieChart } from 'react-minimal-pie-chart';
+
+import postgame from 'public/postgame.png';
+
+const postgameData = {
+	name: 'Player 1',
+	correct: 16,
+	wrong: 8,
+	unanswered: 2,
+};
+
+export default function PostgamePlayerPage() {
+	// Calculate the rank text based on % correct
+	const rank = (() => {
+		const total =
+			postgameData.correct + postgameData.wrong + postgameData.unanswered;
+		const percentCorrect = (100 * postgameData.correct) / total;
+
+		if (percentCorrect === 100) {
+			return 'Perfect!';
+		} else if (percentCorrect >= 90) {
+			return 'Awesome!';
+		} else if (percentCorrect >= 80) {
+			return 'Great!';
+		} else if (percentCorrect >= 70) {
+			return 'Nice!';
+		} else {
+			return 'Good try!';
+		}
+	})();
+
+	return (
+		<main className="w-full h-screen bg-purple-100 flex flex-col items-center justify-center font-extrabold">
+			<div className="w-4/5 h-full flex flex-col items-center justify-center">
+				<Image
+					alt="postgame logo"
+					src={postgame}
+					width={130}
+					className="-mb-4"
+				/>
+
+				<div className="w-fit bg-gray-100 border border-black px-8 py-1 text-3xl z-20 2xl:text-4xl">
+					How you did
+				</div>
+
+				<div className="w-full h-3/4 bg-purple-500 flex items-center justify-center p-8 pt-10 -mt-6 text-xl rounded-3xl z-10 sm:h-2/3 sm:p-12 sm:rounded-xl 2xl:text-3xl">
+					<div className="w-full h-full flex flex-col items-center justify-center gap-8 bg-gray-100 bg-opacity-50 rounded-xl shadow-heavy sm:flex-row sm:gap-16 sm:w-11/12 2xl:gap-20">
+						<PieChart
```

Comment:
The css for the PieChart is mostly good, but there are sizes before sm where it essentially vanishes.

Reply:
Good catches, I'll definitely fix them!